recipe/setup_py.patch

```diff
@@ -0,0 +1,15 @@
+ tools/pythonpkg/setup.py | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/tools/pythonpkg/setup.py b/tools/pythonpkg/setup.py
+--- a/tools/pythonpkg/setup.py
++++ b/tools/pythonpkg/setup.py
+@@ -286,7 +286,7 @@ def setup_data_files(data_files):
+         'duckdb-stubs'
+     ],
+     include_package_data=True,
+-    setup_requires=setup_requires + ["setuptools_scm<7.0.0", 'numpy>=1.14', 'pybind11>=2.6.0'],
++    setup_requires=setup_requires + ["setuptools_scm", 'numpy>=1.14', 'pybind11>=2.6.0'],
```

Comment:
if you update from the `main` branch you can skip this patch since I added `setuptools` to the `build` environment.